metric/config.go

```diff
@@ -31,6 +31,8 @@ func (cfg MeterConfig) SchemaURL() string {
 }
 
 // MeterOption is an interface for applying Meter options.
+//
+// Warning: methods may be added to this interface in minor releases.
```

Comment:
This, and other interfaces that include unexported methods, cannot be implemented outside of the package that contains it and thus probably doesn't need this comment.

Reply:
@Aneurysm9 Done ✅